### PR TITLE
Timestamp data add in tables of Events tab

### PR DESCRIPTION
**Pull request by JuanGarriuz**
### Description

Added timestamp columns in Events tab.
 
### Issues Resolved

- https://github.com/wazuh/wazuh-dashboard-plugins/issues/6791

### Evidence

image

#### Checks all Modules > Events render timestamp column

- [ ] Configuration Assessment
- [ ] Malware Detection
- [ ] File Integrity Monitoring
- [ ] Vulnerability Detection
- [ ] Virus Total
- [ ] PCI DSS
- [ ] GDPR
- [ ] HIPAA
- [ ] NIST 800-53
- [ ] TSC
- [ ] Docker
- [ ] Amazon Web Services
- [ ] Google Cloud
- [ ] GitHub
- [ ] Office 365

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Added HAProxy helper settings to cluster configuration [#6653](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6653)
 - Added ability to open the report file or Reporting application from the toast message [#6558](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6558)
 - Added support for agents to Office 365 [#6558](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6558)
+- Added timestamp columns in events tabs [#6792](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6792)
```
Comment (Desvelao) [inline CHANGELOG.md:21]: **nitpick**: move the pull request reference to the `Removed embedded discover` entry

**Comment by Desvelao** [comment]
# Test

<details>

<summary>Checks all Modules > Events render timestamp column</summary>

Configuration Assessment
 
 TODO
 
:green_circle:  Malware Detection
 
image

:green_circle: File Integrity Monitoring
 
image

:green_circle: Vulnerability Detection
image

:green_circle: Virus Total
 
image

:green_circle: PCI DSS
 image

:green_circle: GDPR
 image

:green_circle: HIPAA
 image

:green_circle: NIST 800-53
 
image

:green_circle: TSC
 
image

:green_circle: Docker
 
image

:green_circle: Amazon Web Services
 
image

:green_circle: Google Cloud
 
image

:green_circle: GitHub
 
image

:green_circle: Office 365
 
image

</details>

**Review by Desvelao — APPROVED** [review]
LGTM!

**Comment by Machi3mfl** [comment]
## Test

#### Checks all Modules > Events render timestamp column

- Configuration Assessment
- Malware Detection
- File Integrity Monitoring
- Vulnerability Detection
- Virus Total
- PCI DSS
- GDPR
- HIPAA
- NIST 800-53
- TSC
- Docker
- Amazon Web Services
- Google Cloud
- GitHub
- Office 365

**Review by Machi3mfl — APPROVED** [review]

**Pull request merged**